include/sway/tree/root.h

```diff
@@ -16,10 +16,44 @@ struct sway_root {
 	struct wlr_output_layout *output_layout;
 
 	struct wl_listener output_layout_change;
+
+	// scene node layout:
+	// - root
+	// 	- staging
+	// 	- interactive
+	// 		- layer shell stuff
+	// 		- tiling
+	// 		- floating
+	// 		- fullscreen stuff
+	// 	- noninteractive
+	// 		- seat stuff
+	struct wlr_scene *root_scene;
+
+	// since wlr_scene nodes can't be orphaned and must always
+	// have a parent, use this staging scene_tree so that a
+	// node always have a valid parent. Nothing in this
+	// staging node will be visible.
+	struct wlr_scene_node *staging;
+
+	// only some layers are supposed to be interactive through input.
+	struct wlr_scene_node *interactive;
+	struct wlr_scene_node *noninteractive;
+
+	struct {
+		struct wlr_scene_node *shell_background;
+		struct wlr_scene_node *shell_bottom;
+		struct wlr_scene_node *tiling;
+		struct wlr_scene_node *floating;
+		struct wlr_scene_node *shell_top;
+		struct wlr_scene_node *fullscreen;
```

Comment:
Hmm, did you forget to remove this?

Reply:
No, fullscreen and fullscreen_global are different things?